pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -2097,6 +2089,44 @@ pub mod pallet {
 			});
 		}
 
+		fn try_remove_avatar_ownership_from(
```

Comment:
Maybe we want to call this `try_mover_avatar_ownership_to_technical_account`?

Reply:
Hmm well both names are fine, since the technical account doesn't own the avatar per se, it mostly holds it, since it cannot do anything with it anyways.